test/contributionreward.js

```diff
@@ -35,9 +35,7 @@ const setup = async function (accounts,orgNativeTokenFee=0) {
    testSetup.genesisScheme = await GenesisScheme.new({gas:constants.GENESIS_SCHEME_GAS_LIMIT});
    testSetup.org = await helpers.setupOrganization(testSetup.genesisScheme,accounts[0],1000,1000);
    testSetup.contributionRewardParams= await setupContributionRewardParams(testSetup.contributionReward,orgNativeTokenFee);
-   //give some tokens to organization avatar so it could register the universal scheme.
-   await testSetup.standardTokenMock.transfer(testSetup.org.avatar.address,30,{from:accounts[1]});
-   var permissions = "0x0000000F";
+   var permissions = "0x00000000";
```

Comment:
Shouldn't the permission be 0x00000001 according to our convention?

Reply:
It is handle by registerScheme so actually there is no need to turn on this bit.

see : 
https://github.com/daostack/Arc/blob/master/contracts/controller/UController.sol#L182
https://github.com/daostack/Arc/blob/master/contracts/controller/Controller.sol#L170